devel/STC/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           github 1.0
+
+github.setup        stclib STC 4.2 v
+revision            0
```

Comment:
missing "github.tarball_from"

Reply:
> missing "github.tarball_from"

Because default tags work, why add extra code :) 

But anyway, I will do this now and rebase.

Reply:
the why for this has been discussed sufficiently now in different PRs..